packages/types/src/oauth.ts

```diff
@@ -109,6 +111,12 @@ export const OAUTH_PROVIDERS: OAuthProviderData[] = [
     name: 'LinkedIn',
     docsUrl: 'https://clerk.com/docs/authentication/social-connection-with-linkedin',
   },
+  {
+    provider: 'linkedin_oidc',
+    strategy: 'oauth_linkedin_oidc',
+    name: 'LinkedIn',
+    docsUrl: 'https://clerk.com/docs/authentication/social-connections/linkedin-oidc',
```

Comment:
This does not exist yet, right? Are we adding it?

Reply:
@desiprisg Yes: https://github.com/clerkinc/clerk-docs/pull/342